src/types.ts

```diff
@@ -23,6 +23,16 @@ export interface ExtendedTranslator extends stardust.Translator {
   language(): string;
 }
 
+export interface Galaxy {
+  translator: ExtendedTranslator;
```

Comment:
is the translator on here? I thought we got it from the useTheme hook?

Reply:
Yes, the translator is from Qlik Sense https://qlik.dev/libraries-and-tools/nebulajs/in-qlik-sense